plugins/plugin-languageserver/che-plugin-languageserver-ide/src/main/java/org/eclipse/che/plugin/languageserver/ide/LanguageServerFileTypeRegister.java

```diff
@@ -59,21 +55,19 @@
     private final HoverProvider                       hoverProvider;
     private final OccurrencesProvider                 occurrencesProvider;
 
-    private final Map<String, String> ext2langId = new HashMap<>();
-
     @Inject
-    public LanguageServerFileTypeRegister(LanguageServerRegistryServiceClient serverLanguageRegistry,
-                                          FileTypeRegistry fileTypeRegistry,
-                                          LanguageServerResources resources,
+    public LanguageServerFileTypeRegister(LanguageServerRegistryServiceClient serverLanguageRegistry, 
+                                          LanguageServerRegistry lsRegistry,
+                                          LanguageServerResources resources, 
                                           EditorRegistry editorRegistry,
-                                          OrionContentTypeRegistrant contentTypeRegistrant,
+                                          OrionContentTypeRegistrant contentTypeRegistrant, 
                                           OrionHoverRegistrant orionHoverRegistrant,
                                           OrionOccurrencesRegistrant orionOccurrencesRegistrant,
-                                          LanguageServerEditorProvider editorProvider,
+                                          LanguageServerEditorProvider editorProvider, 
                                           HoverProvider hoverProvider,
-                                          OccurrencesProvider occurrencesProvider) {
+                                          OccurrencesProvider occurrencesProvider) { 
```

Comment:
@tsmaeder you have to be consistent, sometimes you add trailing spaces, sometimes not. Can you remove them ?